nilearn/plotting/find_cuts.py

```diff
@@ -76,6 +76,8 @@ def find_xyz_cut_coords(img, mask=None, activation_threshold=None):
         activation_threshold = fast_abs_percentile(my_map[my_map != 0].ravel(),
                                                    80)
     mask = np.abs(my_map) > activation_threshold - 1.e-15
+    if mask.sum() == 0:
```

Comment:
Could you add a comment ? I'm not sure I understand what you do here. 
+ a test...